.mergify.yml

```diff
@@ -0,0 +1,11 @@
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - "#approved-reviews-by>=1"
+      - author=scalameta-bot
+      - "#check-success>=22"
```

Comment:
Any chance to make it on all jobs? We might forget to modify it at some point

Reply:
Maybe we can define another github action that checks all other (important) tests passing.

Reply:
We can follow up later with that :+1: